apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_utils/getContractPageSidebarLinks.ts

```diff
@@ -0,0 +1,146 @@
+import type { SidebarLink } from "@/components/blocks/Sidebar";
+import type { ContractPageMetadata } from "./getContractPageMetadata";
+
+// todo - separate out "extensions" under extension group and not all links under same group
+
+export function getContractPageSidebarLinks(data: {
+  metadata: ContractPageMetadata;
+  contractAddress: string;
+  chainSlug: string;
+}) {
+  const layoutPrefix = `/${data.chainSlug}/${data.contractAddress}`;
+
+  const sidebarLinks: SidebarLink[] = [
+    {
+      group: "General",
+      links: [
+        {
+          label: "Overview",
+          href: layoutPrefix,
+          exactMatch: true,
+        },
+        {
+          label: "Modules",
+          href: `${layoutPrefix}/modules`,
+          hide: !data.metadata.isModularCore,
+          exactMatch: true,
+        },
+        {
+          label: "Code Snippets",
+          href: `${layoutPrefix}/code`,
+          exactMatch: true,
+        },
+        {
+          label: "Explorer",
+          href: `${layoutPrefix}/explorer`,
+          exactMatch: true,
+        },
+        {
+          label: "Events",
+          href: `${layoutPrefix}/events`,
+          exactMatch: true,
+        },
+        {
+          label: "Embed",
+          href: `${layoutPrefix}/embed`,
+          hide: data.metadata.embedType === null,
```

Comment:
instead of `hiding` it can we just not add it? or filer it out before returning/using it?

Reply:
Changed it